src/base/bittorrent/session.cpp

```diff
@@ -2063,6 +2063,20 @@ bool Session::addTorrent_impl(CreateTorrentParams params, const MagnetUri &magne
             patchedFastresumeData.replace("6:pausedi0e", "6:pausedi1e");
             patchedFastresumeData.replace("12:auto_managedi0e", "12:auto_managedi1e");
 
+            // converting relative save_path to absolute
+            int start = patchedFastresumeData.indexOf("9:save_path") + 11;
+            if (start > -1) {
+                int end = patchedFastresumeData.indexOf(":", start);
+                int len = patchedFastresumeData.mid(start, end - start).toInt();
+                if (len > 0) {
+                    QByteArray oldPath = patchedFastresumeData.mid(end + 1, len);
+                    QByteArray newPath = Profile::instance().fromPortablePath(Utils::Fs::toUniformPath(fromLTString(oldPath))).toUtf8();
```

Comment:
These two variables can be const, isn't it? Also please choose more meaningful names.

Reply:
Added `const` to all variables and renamed some of them. Feel free to write here names for variables that you want to see and I will change them.